arches_her/media/js/views/components/workflows/upload-document-step.js

```diff
@@ -115,6 +109,23 @@ define([
             params.resourceid(ko.unwrap(this.consultationResourceId));
             params.workflow.resourceId(ko.unwrap(this.consultationResourceId));
     
+            params.defineStateProperties = function(){
```

Comment:
defineStateProperties should probably just be removed here

Reply:
The branch is updated the most recent master as of 12/10 and some minor fixe was added.

Reply:
This is ready to be reviewed again.